CHANGELOG.md

```diff
@@ -35,6 +35,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-3024**
```

Comment:
We need a changelog note re: granule updates and the db package translate update.

Reply:
Added a note: https://github.com/nasa/cumulus/pull/3082/commits/0991d0104b12a620e123de9b1868931de740351a

There's a lot of refactoring going on but I think that covers the functional changes. Lemme know if you agree.

Reply:
I think that's mostly right, I was looking for an explicit call-out on the changes to the exported translate method/addition of the new method.